sharepoint/sharepoint-ps/sharepoint-online/Get-SPOSiteFileVersionBatchDeleteJobProgress.md

```diff
@@ -0,0 +1,69 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spositefileversionbatchdeletejobprogress
+applicable: SharePoint Online
+title: Get-SPOSiteFileVersionBatchDeleteJobProgress
+schema: 2.0.0
+author: msjennywu
+ms.author: jennywu
+ms.reviewer:
+manager: seanmc
+---
+
+# Get-SPOSiteFileVersionBatchDeleteJobProgress
+
+## SYNOPSIS
+
+> [!NOTE]
+> This feature is part of the version history controls preview. If your tenant is not part of the preview or the feature has not rolled out to your tenant, you will get an error when trying to run this cmdlet.
+
+Gets the progress of a trim job for a site collection.
+
+## SYNTAX
+
+```powershell
+Get-SPOSiteFileVersionBatchDeleteJobProgress [-Identity] <SpoSitePipeBind> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Gets the progress of a trim job for a site collection.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Get-SPOSiteFileVersionBatchDeleteJobProgress -Identity https://contoso.sharepoint.com/sites/site1
+```
+
+Example 1 gets the progress of a trim job for a site collection.
```

Comment:
Example 1 Tracks progress of version trim job for site 'https://contoso.sharepoint.com/sites/site1'.